### ISSUE TITLE
Error "Cannot convert undefined or null to object" after upgrade from 2.6.3 to 3.0.x

### ISSUE THREAD

**Issue by abenhamdine** [nuxt/webpackbar#30]
### Version

[v3.0.3](https://github.com/webpackbar/releases/tag/v3.0.3)

### Reproduction link

[https://github.com/nuxt/webpackbar](https://github.com/nuxt/webpackbar)

### Steps to reproduce

Build a project with create-react-app, webpack 3 and react-app-rewired, with following config-override : 
```js
module.exports = {
	webpack: function (config, env) {

		config.plugins = (config.plugins || []).concat([
			new WebpackBar({
				profile: true,
			})
		])

		return config

	}
}
```

Or way to reproduce the issue : run tests for ant-design project (https://github.com/ant-design/ant-design) with `npm run dist`.
See CI failure here : https://circleci.com/gh/ant-design/ant-design/24699?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link.  
The upgrade has been reverted since by this commit : https://github.com/ant-design/ant-design/commit/5204b45e01e91026c8cc63dba343b1c67ca8f7a0

### What is expected ?

No error

### What is actually happening?

Error Cannot convert undefined or null to object

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/webpackbar/issues/c23">#c23</a>)</em></sub></div>

**Comment by pi0** [comment]
Thanks, @abenhamdine for reporting this important inconsistency. The main fix was 8f0085a3510f19c60d8e0620c5946c8d08e3d08e. But after that I tried to make several improvements to ensure more stability and published new changes :) Please feel reporting back if it works for you or not.

**Comment by abenhamdine** [comment]
version 3.1.2 fixed that issue and works perfectly :+1: 
Thx for your responsiveness and for this great module !
